README.adoc

```diff
@@ -152,12 +152,16 @@ Congratulations! You built and deployed a Spring Boot app to Azure Spring Apps.
 
 IMPORTANT: Don't forget to delete the Azure resources created if no longer needed. 
 
+== Run with AZD
+You can also use (https://learn.microsoft.com/azure/developer/azure-developer-cli/overview)[the Azure Developer CLI (azd)] to get you up and running this application on Azure Spring Apps quickly. Azure Developer CLI is an open-source tool that accelerates the time it takes for you to get your application from local development environment to Azure.
+
+For how to run it with azd, you can refer to https://learn.microsoft.com/azure/spring-apps/quickstart[Deploy your first application to Azure Spring Apps]
```

Comment:
Update this to a proper markdown link...

Reply:
> YOu don't want it to be a proper hyperlink?

hey @jongio , in my opinion, I agreed with @yiliuTo . The adoc way looks better in this repo.